LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright 2023 Taiko Labs
```

Comment:
Do we need to update this each year? I think the copy right year is the year we started, not the current year.

Reply:
5ed34aa06dea293dfebebd2c6cbff0ae797d4ed8

apparently doesn't matter. reverted. copyright year and license protections are separate anyways, so this is fine.